pipelines.md

```diff
@@ -63,7 +63,9 @@ When the Kabanero operator activates the CRD, it associates the pipelines in the
 
 ### Updating default tasks and pipelines
 
-The default tasks and pipelines can be updated by forking the Kabanero Pipelines repo and editing the files under `pipelines/incubator`.  The easiest way to generate the archive for use by the Kabanero CRD is to run the [package.sh](https://github.com/kabanero-io/kabanero-pipelines/blob/master/ci/package.sh) script. The script generates the archive file with the necessary pipeline artifacts and a `manifest.yaml` file that describes the contents of the archive. Alternatively, you can run the Travis build against a release of your pipelines repo, which also generates an archive file and a `manifest.yaml` file.
+The default tasks and pipelines can be updated by forking the Kabanero Pipelines repo and editing the files under `pipelines/incubator`.  The easiest way to generate the archive for use by the Kabanero CRD is to run the [package.sh](https://github.com/kabanero-io/kabanero-pipelines/blob/master/ci/package.sh) script. The script generates the archive file with the necessary pipeline artifacts and a `manifest.yaml` file that describes the contents of the archive.  Copy the package.sh file to the root directory of your pipelines project and run it.  It will generate the pipelines archive file under `ci/assests`.
```

Comment:
Change
Copy the package.sh file
to
Copy the `package.sh` file

Change
It will generate
to
It generates